aspnetcore/fundamentals/host/hosted-services.md

```diff
@@ -80,7 +80,7 @@ The <xref:Microsoft.Extensions.Hosting.IHostedService> interface defines two met
 
   The cancellation token has a default five second timeout to indicate that the shutdown process should no longer be graceful. When cancellation is requested on the token:
 
-  * Any remaining background operations that the app is performing should be aborted.
+  * Any remaining background operations that the app is performing in `ExceuteAsync` should be aborted.
```

Comment:
Revert, this is still IHostedService, not BackgroundService. An IHostedService could do background operations any way it wants.